### PR TITLE
Add tooltips to Rendering Debugger (HDRP & URP)

### PR DESCRIPTION
**Pull request by arttu-peltonen**
### Purpose of this PR
Rendering Debugger window was missing tooltips for both HDRP and URP. This PR adds them.

---
### Testing status
Checked that the tooltips are visible in the editor.

### CODE REVIEW

**Review by erikabar — APPROVED** [review]

**Comment by arttu-peltonen** [comment]
This is still waiting on some HDRP tooltips to be written. @Vic-Cooper please let me know when it's ready from your end, and I'll add them too :)

File: com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs [hunk 1]
```diff
@@ -178,6 +179,23 @@ internal float AlbedoSaturationTolerance
...
 
+        static class Strings
+        {
+            public const string AlbedoSettingsContainerName = "Albedo Settings";
+            public const string MetallicSettingsContainerName = "Metallic Settings";
+
+            public static readonly NameAndTooltip MaterialOverride = new() { name = "Material Override", tooltip = "Use the drop-down to select a Material property to visualize on every GameObject on screen." };
+            public static readonly NameAndTooltip VertexAttribute = new() { name = "Vertex Attribute", tooltip = "Use the drop-down to select a 3D GameObject attribute, like Texture Coordinates or Vertex Color, to visualize on screen." };
+            public static readonly NameAndTooltip MaterialValidationMode = new() { name = "Material Validation Mode", tooltip = "Debug and validate material properties." };
+            public static readonly NameAndTooltip ValidationPreset = new() { name = "Validation Preset", tooltip = "Validate through the use of a predetermined list of preset surfaces and inputs based on real world surfaces." };
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs:190]: I think "preset" negates "predetermined" - all presets are predetermined :D. Suggestion:

"Validate using a list of preset surfaces and inputs based on real-world surfaces."

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Suggested some changes to the tooltips :)

**Review by JulienIgnace-Unity — APPROVED** [review]

**Review by lukaschod — APPROVED** [review]

**Review by ellioman — APPROVED** [review]

File: com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs [hunk 2]
```diff
@@ -178,6 +179,23 @@ internal float AlbedoSaturationTolerance
...
+            public static readonly NameAndTooltip AlbedoMinLuminance = new() { name = "Min Luminance", tooltip = "Any values set below this field will be considered invalid and will appear red on screen." };
+            public static readonly NameAndTooltip AlbedoMaxLuminance = new() { name = "Max Luminance", tooltip = "Any values set above this field will be considered invalid and will appear blue on screen." };
+            public static readonly NameAndTooltip AlbedoHueTolerance = new() { name = "Hue Tolerance", tooltip = "Validate a material based on a specific hue." };
+            public static readonly NameAndTooltip AlbedoSaturationTolerance = new() { name = "Saturation Tolerance", tooltip = "Validate a material based on a specific Saturation." };
+            public static readonly NameAndTooltip MetallicMinValue = new() { name = "Min Value", tooltip = "Any values set below this field will be considered invalid and will appear red on screen." };
+            public static readonly NameAndTooltip MetallicMaxValue = new() { name = "Max Value", tooltip = "Any values set above this field will be considered invalid and will appear blue on screen." };
```
[the remark this answers is not in the document]
Comment (arttu-peltonen) [reply com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs]: Thanks! Fixed this and all others below as requested.

**Pull request merged**